### DISCUSSION TITLE
Statistics on delays

### DISCUSSION THREAD

**Discussion by juengerjustus** [Traewelling/traewelling#2392] [Ideas / Feature Requests]
I would like to see the introduction of a new statistic. 
The statistics would revolve around delays. On the one hand, the reasons for the delay could be collected (and generally displayed in posts/check ins) if the DB API allows this and rank them based on their number.

However, it would mainly be desirable to have a statistic that shows the minutes of delay. E.g.: planned time on public transport vs. actual time spent on public transport or "You have X minutes/hours/... delay".
This could be further expanded with these delay statistics for the various means of transport and journey purposes.

**Comment by marhei** [comment]
Seems to be a duplicate of #2021
